features/accommodations/useAccommodationsMap.ts

```diff
@@ -0,0 +1,27 @@
+import { useQuery } from "react-query";
+import axios from "axios";
+import { createEntitiesMap } from "../../core/utils/map";
+
+type AccommodationInfo = {
+  id: string;
+  title: string;
+};
+
+export function useAccommodationsMap(accommodationIds: string[], dependencies: any[] = []) {
+  const { data, isFetching, error } = useQuery(
+    [accommodationIds, ...dependencies],
+    () =>
+      axios.get(
+        `/catalogService/api/accommodations?${accommodationIds?.map((id) => "ids=" + id).join("&")}`
+      ),
+    {
+      enabled: dependencies?.reduce((acc, dep) => acc && !dep, true) && accommodationIds?.length,
```

Comment:
Add !! because accommodationIds?.length is not evaluates as boolean and I am getting error.